### ISSUE TITLE
Is this working? 

### ISSUE THREAD

**Issue by Marow** [rkanadev/go-dubaicoinrk#1]
There is no Wiki and readme are much things copy from ethereum?

**Comment by rkanadev** [comment]
Yes, there is no wiki yet - we will be working on it in the future. Right now all basic commands and other are based on eth.
